about.html

```diff
@@ -1,73 +1,86 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-
-  <title>Sprint Challenge - About</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-</head>
-
-<body>
+  <head>
+    <meta charset="utf-8">
+    <title>Sprint Challenge - About</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet">
+    <link href="css/index.css"
```

Comment:
Think about breaking up your code into different css files for each html file. It makes it a lot easier to work with.